Src/Plugins/AzureDevOpsConfiguration/ServiceConnectionSecurityValidators/AZC102_190.DoNotGrantAllPipelinesAccessValidator.cs

```diff
@@ -88,8 +89,7 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
                 }
 
                 HttpClient httpClient = CreateOrRetrieveCachedHttpClient();
-
-                string apiUri = string.Format(PipelinePermissionAPI, organization, project, serviceConnectionId);
```

Comment:
>string [](http://example.com/codeflow?start=16&length=6)

in general, we should hunt down string.format utilization in favor of interpolated strings. I did some of that here but tied off the work in order to close the PR (I left test code unimproved in some cases, for example).